README.md

```diff
@@ -219,6 +219,19 @@ postcss()
 
 This is mainly for use by postcss runners that implement file watching.
 
+## Usage in Deno
+
+This plugin is available for [Deno](https://deno.land/):
+
+```js
+import postcss from "https://deno.land/x/postcss/mod.js";
+import postcssImport from "https://deno.land/x/postcss_import/mod.js";
+
+const result = await postcss([postcssImport]).process(css);
+```
+
+Note that the Deno version has not support for module resolution like Node (packages installed with npm or folders with a `package.json` file).
```

Comment:
```suggestion
Note that the Deno version does not support Node-like module resolution (packages installed with npm or folders with a `package.json` file).
```
Clarified sentence. 

Just so I understand, that means that in deno, you can only use imports like `@import "./path/to/file.css"`?

Reply:
Node version uses the `resolve` package as a dependency for all resolutions. I didn't found any deno package that does that, so I've replicated the functionality here: https://github.com/postcss/postcss-deno-import/blob/master/deps.js#L11

For now, it have the following support:

- `@import "./styles.css"` - Obviously
- `@import "styles.css"` - Search the file as relative path (like`./styles.css`) and in all `options.paths`
- `@import "styles"` - Detect `./styles.css` or `./styles/index.css` in the relative path and all `options.paths`.

### What is missing?

It does not load `package.json` files to detect the file in the `style` or `main` fields. This feature could be added but due this is a Deno package, I don't think it makes sense load Node packages here, so I decided don't complicate the code.